exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetRecordWriter.java

```diff
@@ -423,6 +459,34 @@ public void writeField() throws IOException {
     }
   }
 
+  @Override
+  public FieldConverter getNewRepeatedListConverter(int fieldId, String fieldName, FieldReader reader) {
+    return new RepeatedListParquetConverter(fieldId, fieldName, reader);
+  }
+
+  public class RepeatedListParquetConverter extends FieldConverter {
+    private final FieldConverter converter;
+
+    RepeatedListParquetConverter(int fieldId, String fieldName, FieldReader reader) {
+      super(fieldId, fieldName, reader);
+      converter = EventBasedRecordWriter.getConverter(ParquetRecordWriter.this, 0, "", reader.reader());
+    }
+
+    @Override
+    public void writeField() throws IOException {
+      consumer.startField(fieldName, fieldId);
+      consumer.startField("list", 0);
```

Comment:
Should we use constant?

Reply:
I'm also not sure whether constant is necessary here:))) It's just passing index of field within group of one element. I'm afraid having improperly named constant may create only confusion here.

Reply:
I don't have strong opinion here, you may leave as it, the main reason why I have written this comment is that I saw the same string repeating several times.

Reply:
ok, I'll add few constants to ```FieldConverter```.